client/blocks/reader-feed-header/index.jsx

```diff
@@ -50,12 +52,19 @@ class FeedHeader extends Component {
 		const siteTitle = getSiteName( { feed, site } );
 		const siteUrl = getSiteUrl( { feed, site } );
 		const isEmailBlocked = userSettings.getSetting( 'subscription_delivery_email_blocked' );
+		const siteId = site && site.ID;
 
 		const classes = classnames( 'reader-feed-header', {
 			'is-placeholder': ! site && ! feed,
 			'has-back-button': showBack,
 		} );
 
+		const notificationSettings = config.isEnabled( 'reader/new-post-notifications' ) ? (
```

Comment:
before we would never create these elements if `siteId` was falsey. Moving it out here changes that a bit... Can these elements handle a null siteId?